app/client/aclui/AccessRules.ts

```diff
@@ -1809,6 +1817,9 @@ class ObsRulePart extends Disposable {
   private _warnInvalidColIds(colIds?: string[]) {
     if (!colIds || !colIds.length) { return false; }
     const allValid = new Set(this._ruleSet.getValidColIds());
+    // Don't check column validity if we don't have any.
+    // For example, the seed rule has no columns.
+    if (allValid.size === 0) { return false; }
```

Comment:
I think there are other cases of no-columns:
- the default rule (fallback for all tables). I think it would be hard to impossible to avoid it being an invalid rule then.
- the special rules. Using columns there would probably make them invalid too.

Can we maybe target seed rules more specifically here?

Reply:
Ok, will bring back some code for that, this just felt simpler but you're right, it is too broad.